### ISSUE TITLE
3 Button Navigation - Random no function.

### ISSUE THREAD

**Issue by Goopdegooo** [GrapheneOS/os-issue-tracker#3408]
Is anyone experiencing any issues with 3 button navigation?

I haven't been able to use the home button (circle) on occasions, it just doesn't function. I have had times where i can't use it at all and needed to close all apps in the task bar on occasion because it could not get out of the menu using the square button either.  I'm wondering if anyone else has seen this?

**Comment by Goopdegooo** [comment]
Managed to get it reordered..

https://github.com/GrapheneOS/os-issue-tracker/assets/147979249/7c166f55-3db6-4181-bca5-f754e47ae5b0

**Comment by p0wl90** [comment]
Yes I have the same issue since a while on two different Pixel 8 phones. There are two threads in the discussion forum about it:

* [3-button navigation freezes](https://discuss.grapheneos.org/d/9928-3-button-navigation-freezes) 
* [3 button navigation does not work after update](https://discuss.grapheneos.org/d/8520-3-button-navigation-does-not-work-after-update)

There are some possible workarounds in these threads and but not all of them seem to work for everyone:

* Disabling user profiles 
* Restarting the device may help
* Setting "Animator duration scale to "Off"
* Force-Stopping the default launcher
* It might be related to custom launchers (e.g. Nova) but I could not find any reports about this behavior in Nova launcher community

Please tackle this bug since it is really the only thing not working properly with GrapheneOS but is a bug that really impacts everyday user experience.
